app/build.gradle

```diff
@@ -1,33 +1,34 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "27.0.3"
-
+    compileSdkVersion 28
+    buildToolsVersion "29.0.3"
 
     defaultConfig {
         applicationId "com.phikal.regex"
         minSdkVersion 16
-        targetSdkVersion 24
+        targetSdkVersion 28
         versionCode 5
         versionName "1.3.1"
     }
+
     buildTypes {
         release {
             minifyEnabled true
-            shrinkResources true
```

Comment:
Is there a downside to this I don't know of?

Reply:
Nope, I just find that it makes debugging particularly difficult if a crash occurs (minify doesn't help, either, but that's fine).

Reply:
But won't it only be shrunk in release builds? It's unusual for people to use them for debugging. I've never really understood Gradle, so forgive my ignorance.

Reply:
It's ok, you're right. I can't actually remember why I removed this haha.